core/src/gui/widgets/combo_as_slider.cpp

```diff
@@ -0,0 +1,30 @@
+#include <gui/widgets/combo_as_slider.h>
```

Comment:
I wrote this for usage in rx888_source - for selecting attenuation mostly (it has small step so many values)

Reply:
Imo a stepper slider widget would be better for that applciation.

Reply:
This is stepped slider, but with custom label.

The main idea was that this will be combobox (selecting some values) but in form of slider (because of the occupied space)